locales/en-US/app.ftl

```diff
@@ -143,9 +143,9 @@ shareCopy = Copy
 
 eolMessage = <strong>This experiment is ending on {$completedDate}</strong>.<br/><br/>After then you will still be able to use {$title} but we will no longer be providing updates or support.
 
-restartRequiredHeader = Test Pilot Installed!
-restartRequiredFromLanding = Restart Firefox to choose your features.
-restartRequiredFromExperiment = Restart Firefox to enable {$experimentTitle}.
+restartRequiredSubHeader = Almost done . . .
```

Comment:
@chuck Technically, the &hellip; character entity is not correct punctuation in this case: 

http://www.thepunctuationguide.com/ellipses.html